.changeset/sour-cobras-share.md

```diff
@@ -0,0 +1,11 @@
+---
+"wrangler": patch
+---
+
+feat: autogenerated config from dash
+
+Makes `wrangler init`'s `--from-dash` option pull in data from Cloudflare's dashboard to generate a wrangler.toml file.
```

Comment:
```suggestion
Makes `wrangler init`'s `--from-dash` option pull in data from Cloudflare's dashboard to generate a wrangler.toml file populated with configuration from an existing Worker, as well as downloading the code of the existing Worker.
```

Reply:
> populated with configuration from an existing Worker, as well as downloading the code of the existing Worker.

This PR only generates the config, a previous PR generated the code